engine/src/test/java/org/hibernate/validator/test/internal/util/StringHelperTest.java

```diff
@@ -87,6 +88,12 @@ public void decapitalizeShouldReturnDecapizalizedWord() {
 		assertEquals( StringHelper.decapitalize( "Giraffe" ), "giraffe" );
 	}
 
+    @Test
+    public void decapitalizeShouldReturnDecapizalizedWordOnTurkishLocale() {
+        Locale.setDefault(new Locale("tr", "TR"));
+        assertEquals( StringHelper.decapitalize( "IsIsolationLevelGuaranteed" ), "isIsolationLevelGuaranteed" );
```

Comment:
Could you get the default locale before changing it and restore it after the test? It would be cleaner to let the state consistent after the test.